ssr/src/state/canisters.rs

```diff
@@ -218,17 +214,19 @@ impl<const A: bool> Canisters<A> {
         }
         #[cfg(not(any(feature = "local-bin", feature = "local-lib")))]
         {
-            use std::collections::HashSet;
-            // TODO: this is temporary
-            let blacklisted =
-                HashSet::from([Principal::from_text("rimrc-piaaa-aaaao-aaljq-cai").unwrap()]);
-            let orchestrator = self.orchestrator().await;
-            Ok(orchestrator
-                .get_all_available_subnet_orchestrators()
-                .await?
-                .into_iter()
-                .filter(|subnet| !blacklisted.contains(subnet))
-                .collect())
+            let temp_user_idx = Principal::from_text("znhy2-2qaaa-aaaag-acofq-cai").unwrap();
+            return Ok(vec![temp_user_idx]);
+            // use std::collections::HashSet;
+            // // TODO: this is temporary
+            // let blacklisted =
+            //     HashSet::from([Principal::from_text("rimrc-piaaa-aaaao-aaljq-cai").unwrap()]);
+            // let orchestrator = self.orchestrator().await;
+            // Ok(orchestrator
+            //     .get_all_available_subnet_orchestrators()
+            //     .await?
+            //     .into_iter()
+            //     .filter(|subnet| !blacklisted.contains(subnet))
+            //     .collect())
```

Comment:
Do we still need this?

Reply:
Is it safe to uncomment again?

Reply:
no I meant can we not remove this completely?

Reply:
> no I meant can we not remove this completely?

We would need to add it back in the future no?
Or does this user index load balance across all subnets

Reply:
ah just noticed #446 lets merge this first